### PR TITLE
[Localization] Translate For `local\zh-CN\about`

### PR DESCRIPTION
**Pull request by ghost**
All files there are fully translated into Chinese.

### CODE REVIEW

**Comment by willin** [comment]
perhaps u may remove useless whitespaces like: 

```
[ 商标政策 ]
```

**Comment by ghost** [comment]
@willin ：OK, fixed

**Comment by willin** [comment]
LGTM

File: locale/zh-cn/about/community.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - **贡献者** 任何创建或评论议题，或者发送 PR 的个人。
+  - **合作者** 被授予直接可以向 Node 仓库写入权限的贡献者。
+  - **观察者** 指任意一个个人，他（被）要求参与社区委员会的会议。这也是他走向会员的第一步。
+  - **会员** 符合参与要求，具备投票权利的个人，由社区委员会投票选举产生。
+
+欲了解目前的社区委员会成员，请参考 [README.md](https://github.com/nodejs/community-committee)。
+
+## 贡献者与合作者
+
+发展壮大 Node.js 社区是社区委员会的使命。当你读到这时，你其实已经是社区的一份子了——同时 Node.js 社区的一份子。我们热爱你的帮助！
```
Comment (amorist) [inline locale/zh-cn/about/community.md]: > 同时 Node.js 社区的一份子
同时是或者同时成为 比较通顺

Comment (ghost) [reply locale/zh-cn/about/community.md]: OK, Fixed

**Review by amorist — APPROVED** [review]

**Comment by fhemberger** [comment]
Thanks everyone!

**Pull request merged**